Tests/MusicXMLTests/ScoreTests/Reve/manifest.plist

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Doesn't have to be in this PR, but I wonder if we could have configurability around whether or not we want to `assert` parseability / validity of a given test case.

I would prefer to have as many tests as possible `assert` that they don't break the parser. Maybe we could default to `true`, but if there is a new test (suite) which might be flakey for a little while, we could just log the failure?

Reply:
That's good idea